beacon_node/execution_layer/src/engine_api/http.rs

```diff
@@ -583,8 +585,19 @@ impl CapabilitiesCacheEntry {
     }
 }
 
+// used to house the id of the next jsonrpc request and the status of the ws intialization and router since we need to syncronize them all
+pub struct WsPackage {
+    id: u64,
+    status: u8,
+    wsrouter: Option<WsRouter>,
+}
+
+
+// should be initialized: 0: ws needs to be initialized, 1: ws is initialized 2: no need
```

Comment:
We can make this a bit more explicit with an `enum`. It will still be represented as a `u8`.

```rust
pub enum WsStatus {
    Uninitialized,
    Initialized,
    Disabled
}
```

Reply:
added

Reply:
thanks